### ISSUE TITLE
Is it dead?

### ISSUE THREAD

**Issue by vimmerru** [rg35xx-cfw/Koriki#58]
### What happened?

Hi,

I see no any commits for a lot of time and a lot of commits behind original koriki work. Is the project dead?

**Comment by acmeplus** [comment]
https://ko-fi.com/post/Simplermenu-H2H0P654W

**Comment by cesarvieiira** [comment]
> https://ko-fi.com/post/Simplermenu-H2H0P654W

what does it means? Is it dead? Thats the new name of the project, thats another project?

**Comment by acmeplus** [comment]
Yes, that's a different frontend for the same device (and many other devices).

There is an interim configuration update for koriki 0.9.1:
https://github.com/rg35xx-cfw/Koriki/releases/tag/koriki_rg35xx_v0.9.2-test

**Comment by Lalarian** [comment]
> Yes, that's a different frontend for the same device (and many other devices).
> 
> There is an interim configuration update for koriki 0.9.1: https://github.com/rg35xx-cfw/Koriki/releases/tag/koriki_rg35xx_v0.9.2-test

Where and by which name are we gooing to find future firmwares for the RG35xx then?
Thanks! :)
